### ISSUE TITLE
Babel fails to compile object spread when using babel-plugin-transform-runtime

### ISSUE THREAD

**Issue by mmakarin** [babel/babel#3011]
Error
```Cannot resolve module 'babel-runtime/helpers/extends'```

Code
```
var o = {
    foo: "bar"
};
var o2 = {
    "a": 1,
    ...o
};
```

babel-loader config
```
presets: ["es2015", "stage-2", "react"],
plugins: ["transform-runtime"]
```

Package versions
```
"babel-core": "^6.1.4",
"babel-loader": "^6.1.0",
"babel-plugin-transform-runtime": "^6.1.4",
"babel-preset-es2015": "^6.1.4",
"babel-preset-react": "^6.1.4",
"babel-preset-stage-2": "^6.1.2",
"babel-runtime": "^6.1.4",
```

**Comment by timnew** [comment]
I assume it is caused by similar reason as issue #3012

**Comment by mmakarin** [comment]
Fixed in 6.1.18
